nautobot/extras/registry.py

```diff
@@ -14,8 +14,8 @@ def __getitem__(self, key):
             raise KeyError(f"Invalid store: {key}")
 
     def __setitem__(self, key, value):
-        if key in self:
-            raise KeyError(f"Store already set: {key}")
+        # if key in self:
+        #     raise KeyError(f"Store already set: {key}")
         super().__setitem__(key, value)
```

Comment:
This was to make the root keys of the registry overwritable - see earlier comment about how it might be feasible to revert this and handle the nav menu root entries differently.